packages/generator/src/generators/model-generator.ts

```diff
@@ -41,7 +41,10 @@ export class ModelGenerator extends Generator<ModelGeneratorOptions> {
 
   // eslint-disable-next-line require-await
   async write() {
-    const schemaPath = path.resolve("db/schema.prisma")
+    const pkgJson = this.fs.readJSON("package.json", {}) as {[key: string]: any}
```

Comment:
Let's use `cwd`:
```
const packageJsonPath = path.join(process.cwd(), "package.json")
const packageJson = fs.readJSONSync(packageJsonPath)
```

Reply:
The reason for using `this.fs.readJSON` was it loads the root path for us
(so i.e. running this inside of `pages` or `app` will work with `this.fs.readJSON`, but won't with `process.cwd()`)

Reply:
I see. it's fine then